src/content/general/source-data/materials/readme.md

```diff
@@ -0,0 +1,15 @@
+This page will serve as a hub for the symbols and properties users can assign to their materials. We will be learning how the name of our material in our 3D editing software is used to find a valid shader tag, how various special properties can be assigned to a surface through a specific symbol, and set how the lightmapper will use our surface during rasterization. Some examples are adding the `^` symbol to the end of a material name (like `vines^`) so it becomes a climbable ladder, or the special name `+sky` which causes the marked surface to render our assigned sky.
+
+This naming convention applies to both [Blender][] and [3ds Max][3dsmax] usage.
+
+# Halo 1
+This section will cover how material names and shader tags are linked and the properties available to you and what they can accomplish.
+Click the following [link][h1-materials] to proceed.
+
+# Halo 2
```

Comment:
These sections are wordy and redundant. It could just be a bulleted list of gang names with a simple introduction saying the convention differs by game